includes/wikia/TemplateClassification.class.php

```diff
@@ -30,6 +38,15 @@ class TemplateClassification {
 		self::TEMPLATE_QUOTE,
 		self::TEMPLATE_NAVBOX,
 		self::TEMPLATE_UNCLASSIFIED,
+		self::TEMPLATE_NAVBOX,
+		self::TEMPLATE_FLAG,
+		self::TEMPLATE_REF,
+		self::TEMPLATE_MEDIA,
+		self::TEMPLATE_DATA,
+		self::TEMPLATE_DESIGN,
+		self::TEMPLATE_NAV,
+		self::TEMPLATE_NOT_ART,
+		self::TEMPLATE_DONTKNOW,
```

Comment:
DONTKNOW is just user facing message fallback on modal for TEMPLATE_UNKNOWN type so shouldn't be visible here